guide/13-managing-arcgis-applications/hub-submodule-intro.ipynb

```diff
@@ -0,0 +1,71 @@
+   "source": [
+    "## The `hub` submodule\n",
+    "\n",
+    "The `hub` submodule within `arcgis.apps` acts as the pythonic interface to [ArcGIS Hub](https://www.esri.com/en-us/arcgis/products/arcgis-hub/resources). It enables automation of several Hub workflows and simplifies the use of the Hub information model by allowing access to Hub objects such as `Initiative`, `Event`, `Site` and `Page`.\n",
+    "\n",
+    "The capabilities supported in the API both match and exceed those available in Hub's UI. It equips users with tools necessary for working with the Sites in their Hub or Enterprise organization. With it, you can create, search for, edit the layout of, and clone sites and pages within the same organization and to different organizations. \n",
+    "\n",
+    "ArcGIS Hub is available with ArcGIS Online in Hub Basic and Hub Premium, depending on your licensing of Hub. Every user of ArcGIS Online has access to Hub Basic, unless upgraded at the Organization level. Users of ArcGIS Enterprise get access to Sites and Pages, similar to Hub Basic users. Once you sign-in to your `GIS`, you can access the Hub capabilities through `gis.hub`, which is the entry point to Hub. As an Enterprise Sites user, the entry point is `gis.sites`. Let's explore this through the architecture diagrams below.\n",
+    "\n",
+    "## Architecture Diagram\n",
+    "\n",
+    "\n",
+    "![image](https://user-images.githubusercontent.com/13968196/213733145-2f5747e9-4555-4440-a2f2-3b3a13608ec8.png)\n",
+    "\n",
+    "\n",
+    "\n",
+    "You can learn more about the functionality supported through the classes in the hub submodule through the links below:\n",
+    "* [Hub](https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#hub): Represents a Hub organization\n",
+    "* [Initiative](https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#initiative): Represents a Hub Premium Initiative\n",
+    "* [Site](https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#site): Represents a Hub or Enterprise Site\n",
+    "* [Page](https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#page): Represents a Page of a Hub or Enterprise Site\n",
+    "* [Event](https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#event): Represents a Hub Premium Event\n",
+    "* Resource Manager classes for managing these objects above:\n",
+    "    * [InitiativeManager](https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#initiativemanager): To manage initiatives\n",
+    "    * [SiteManager](https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#sitemanager): To manage sites\n",
+    "    * [PageManager](https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#pagemanager): To manage pages\n",
+    "    * [EventManager](https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#eventmanager): To manage events\n",
+    "    \n",
```

Comment:
* the `Developer Experience` team that manages the production [Developer Website](https://developers.arcgis.com/) recommends using relative links to other `guides` or `samples` rather than absolute links. They often change plug-ins that can introduce url changes.
  * please remove the `https://developers.arcgis.com` component
  * for example:
    * The text should be `/python/api-reference/arcgis/apps.hub.html#hub` for the reference to `Hub`